src/DB/projects.json

```diff
@@ -3338,5 +3338,23 @@
       "tech": ["ReactJS","HTML","CSS","Material UI","IMDB API"]
     }
   ]
+},
+{
+"github_username": "Avdhesh-Varshney",
+"Social_media": {
+  "gitHub": "https://github.com/Avdhesh-Varshney",
+  "LinkedIn": "https://www.linkedin.com/in/avdhesh-varshney-5314a4233/",
+  "Instagram": "https://www.instagram.com/_._avdhesh_._/",
+  "YouTube": "#",
+  "Twitter": "https://twitter.com/__Avdhesh__"
+},
+"Projects": [
+  {
+    "link": "https://github.com/Avdhesh-Varshney/Dots-And-Boxes-Game",
+    "title": "Dots and Boxes Game",
+    "description": "This game is a multiplayer game. This project is also for those who want to learn JavaScript as this project is a beginner-friendly.",
+    "tech": ["HTML", "CSS", "JAVASCRIPT"]
+  }
```

Comment:
you've added projects before so just add the sample code in the previous one.

Reply:
@priyankarpal please again review this pr